templates/InstallChocolatey.ps1.erb

```diff
@@ -30,13 +30,14 @@ $chocTempDir = Join-Path $env:TEMP "chocolatey"
 $tempDir = Join-Path $chocTempDir "chocInstall"
 if (![System.IO.Directory]::Exists($tempDir)) {[System.IO.Directory]::CreateDirectory($tempDir)}
 $file = Join-Path $tempDir "chocolatey.zip"
+$chocErrorLog = Join-Path $tempDir "chocError.log"
 
 function Download-File {
 param (
   [string]$url,
   [string]$file
  )
-  Write-Host "Downloading $url to $file"
+  Write-Output "Downloading $url to $file"
```

Comment:
I've replaced the final ones of these to write-output in chocolateySetup.psm1 as well.